docs/en/user-manual/deployment_best_practices.md

```diff
@@ -93,7 +93,7 @@ Note the change of `api_key`, `client_secret`, and `priv_key_path` to `secure_ap
 The contents of the private key file is used as the value of `umapi_private_key_data` in the credential store.  This can only be done on platforms other than Windows.  See below for how to secure the
 private key file on Windows.
 
-The credential values will be looked up using the specified key names with the user being the org_id value.
+The credential values will be looked up in the secure store using org_id as the username value and the key names in the config file as the key name.
```

Comment:
should be "settings values in the config file as the key name" (not "key names in the config file")